webextension/popup.js

```diff
@@ -115,15 +115,29 @@ document.addEventListener("DOMContentLoaded", () => {
   }
 
   autosizeTextArea(document.querySelector("[name=description]"));
+  autosizeTextArea(document.querySelector("[data-detail=description]"));
 });
 
 function autosizeTextArea(el) {
-  el.addEventListener("keydown", function() {
-    requestAnimationFrame(() => {
-      el.style.cssText = "height:auto; padding:0";
-      el.style.cssText = "height:" + el.scrollHeight + "px";
-    });
-  });
+  function resize() {
+    el.style.height = "auto";
+    el.style.padding = "0";
+    const popup = document.scrollingElement;
+    const popupKidHeights = Array.map.call(null, popup.childNodes, n => n.clientHeight);
+    const heightOfRest = popupKidHeights.reduce((a, c) => a + (c || 0), 0) - el.clientHeight;
+    const maxHeight = 588 - heightOfRest; // 588px seems to be the max-height of the popup
```

Comment:
Hard coding 588 makes me a bit nervous... is 588 the max-height just for your laptop? Would it be different for an external monitor, for example?

https://searchfox.org/mozilla-central/source/browser/components/extensions/ExtensionPopups.jsm#516-534